daml-lf/transaction-scalacheck/src/main/scala/com/digitalasset/daml/lf/value/ValueGenerators.scala

```diff
@@ -318,7 +318,7 @@ object ValueGenerators {
         chosenValue,
         stakeholders,
         signatories,
-        actingParties,
+        false,
```

Comment:
```suggestion
        controllersDifferFromActors = false,
```
There is too much argument here. It is probably better to use names for all arguments here.

Reply:
I'm not sure whether writing `signatories = signatories` instead of `signatories` communicates much valuable information. If you still think we should do it, sure.

Reply:
Faire enough.